tests/TestCase/ORM/TableTest.php

```diff
@@ -5593,7 +5593,7 @@ public function testSimplifiedFind()
 
         $query = (new Query($this->connection, $table))->select();
         $table->expects($this->once())->method('callFinder')
-            ->with('all', $query, []);
+            ->with('all', $query);
```

Comment:
Should have more tests for the new feature. It will quickly become cement and we'll need tests to help catch future regressions.